modules/gdscript/gdscript_parser.h

```diff
@@ -1265,6 +1265,7 @@ class GDScriptParser {
 		PREC_FACTOR,
 		PREC_SIGN,
 		PREC_BIT_NOT,
+		PREC_POWER,
```

Comment:
Shouldn't it be further up? This sounds like it has lower precedence as e.g. `PREC_ADDITION_SUBTRACTION`.

Reply:
Ah nevermind I read it wrong. "Higher value" obviously means defined _later_ in the enum.

Reply:
No, the higher value here means high priority

Reply:
Indeed, :)